.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Pull Request
+### Description
+Describe your changes
+### Ticket
+Issue ticket number and link
+### Type of change
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Documentation change
+## Checklist
+### Did you remember to take care of the following?
+- [ ] `npm i` – for new NPM dependencies.
```

Comment:
I would add the checking commands like.
`npm run lint` `npm test` and future command `npm test:screenshots`